appengine/go11x/cloudsql/app.yaml

```diff
@@ -0,0 +1,10 @@
+runtime: go111
+
+env_variables:
+  # Replace INSTANCE_CONNECTION_NAME with the value obtained when configuring your
+  # Cloud SQL instance, available from the Google Cloud Console or from the Cloud SDK.
+  # For Cloud SQL 2nd generation instances, this should be in the form of "project:region:instance".
+  CLOUDSQL_CONNECTION_NAME: INSTANCE_CONNECTION_NAME
+  # Replace username and password if you aren't using the root user.
+  CLOUDSQL_USER: root
+  CLOUDSQL_PASSWORD: ''
```

Comment:
Can we put an example password here? to clarify if it's expecting `password` vs `'password'`

Reply:
YAML don't care. Adding a password.